joomla/joomla-cms
# Articles Module: (Re)Introduced dynamic mode as on mod_articles_category

Pull Request for Issue # .

### Summary of Changes
Added the option for dynamic mode as it was in mod_articles_category


### Testing Instructions
Create a module selecting dynamic mode. Create some menu items for different categories and check if the module display the articles of the category dynamically

Inline help:

> Please select the mode you would like to use. If Normal Mode is chosen, then configure the module and it will display a static list of Articles on the menu items you assign the module to. If Dynamic Mode is chosen, then you can still configure the module normally, however now the Category option will no longer be used. Instead, the module will dynamically detect if you are on a Category view and will display the list of articles within that Category. When Dynamic Mode is chosen, it is best to leave the module set to display on all pages, as it will decide to display anything dynamically.


### Actual result BEFORE applying this Pull Request

No dynamic mode present

### Expected result AFTER applying this Pull Request

The module display a list of articles depending on the category from the menu item


### Link to documentations
Please select:
